core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -80,10 +83,11 @@ public class JdbcCatalog extends BaseMetastoreCatalog
   private final Function<Map<String, String>, FileIO> ioBuilder;
   private final Function<Map<String, String>, JdbcClientPool> clientPoolBuilder;
   private final boolean initializeCatalogTables;
+  private final boolean updateCatalogTables;
   private CloseableGroup closeableGroup;
 
   public JdbcCatalog() {
-    this(null, null, true);
+    this(null, null, true, false);
   }
 
   public JdbcCatalog(
```

Comment:
It looks like this isn't being used anymore, so we should probably mark for deprecation.

Reply:
Good point, let me add `@Deprecated` on `JdbcCatalog(arg1,arg2,arg3)`.